patches/react-scripts+3.4.1.patch

```diff
@@ -0,0 +1,75 @@
+diff --git a/node_modules/react-scripts/config/webpack.config.js b/node_modules/react-scripts/config/webpack.config.js
+index 25840d9..291c7a1 100644
+--- a/node_modules/react-scripts/config/webpack.config.js
++++ b/node_modules/react-scripts/config/webpack.config.js
+@@ -420,6 +420,19 @@ module.exports = function(webpackEnv) {
+                       },
+                     },
+                   ],
++                  [
++                    require.resolve('babel-plugin-transform-imports'),
++                    {
++                      '@material-ui/core': {
++                        'transform': '@material-ui/core/esm/${member}',
++                        'preventFullImport': true
++                      },
++                      '@material-ui/icons': {
++                        'transform': '@material-ui/icons/esm/${member}',
++                        'preventFullImport': true
++                      }
++                    }
++                  ]
```

Comment:
Add the Babel plugin to the config, but only for in-house code, not for the packages we depend on